src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,107 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int threshold;
+    private Node<K, V>[] table;
+    private int size = 0;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+        threshold = (int) (table.length * DEFAULT_LOAD_FACTOR);
+    }
+
+    private class Node<K, V> {
```

Comment:
```suggestion
    private static class Node<K, V> {
```
we can make class static, since it doesn't need access to instance of container class